install/update_91_92.php

```diff
@@ -633,29 +607,32 @@ function update91to92() {
       $migration->renameTable("glpi_bookmarks", "glpi_savedsearches");
 
       $migration->addField("glpi_savedsearches", "last_execution_time", "int(11) NULL DEFAULT NULL");
-      $migration->addKey("glpi_savedsearches", 'last_execution_time');
-
       $migration->addField("glpi_savedsearches", "do_count", "tinyint(1) NOT NULL DEFAULT '2' COMMENT 'Do or do not count results on list display; see SavedSearch::COUNT_* constants'");
-      $migration->addKey("glpi_savedsearches", 'do_count');
-
       $migration->addField("glpi_savedsearches", "last_execution_date", "DATETIME NULL DEFAULT NULL");
-      $migration->addKey("glpi_savedsearches", 'last_execution_date');
-
       $migration->addField("glpi_savedsearches", "counter", "int(11) NOT NULL DEFAULT '0'");
+      $migration->migrationOneTable("glpi_savedsearches");
```

Comment:
This is already the case; I've just tested an upgrade from a 9.1 database against current master; all three keys have been created